### PR TITLE
Communications Update

### PR DESCRIPTION
**Pull request by rhicora**
Added distinct access for all comms channels, allowing IDs to be set up with custom comms access. Added back the Comms Officer and the Comms Technician, and uncordoned the comms tower.

Comms personnel have lots of channels but little physical access. SD no longer gets department channels; they should be talking to either a Department Head or the Tower.

Shortwaves will now provide encrypted channels to anyone with the appropriate comm channel access on their ID. Headsets continue to work off chips.

### CODE REVIEW

File: maps/site53/job/jobs.dm [hunk 1]
```diff
@@ -105,8 +108,8 @@
 	supervisors = "the Security Commander"
 	department = "Command"
 	department_flag = COM
-	total_positions = 0
-	spawn_positions = 0
+	total_positions = 1
```
Comment (JakeDaBoss) [inline maps/site53/job/jobs.dm:113]: Do not enable these yet. I have to add some more things.

**Review by JakeDaBoss — CHANGES_REQUESTED** [review]

**Comment by JakeDaBoss** [comment]
You can enable these jobs when the new uniforms and items relevant to these jobs have been added in, lockers have been made and placed for them, spawn points added, final map fixes and the outfits edited to match.

That's the task list for now. I'll handle the mapping part if you handle the coding parts, please.

**Comment by rhicora** [comment]
- Outfits added for Comms Officer and Comms Dispatcher / Technician
- Uniforms relevant to these jobs have been added
- Icons added for the lockers

Still todo (🦁 can take a look tomorrow):

- Place lockers using those sprites on the map
- Add spawn points
- Test Comms Tower

**Comment by JakeDaBoss** [comment]
Do not merge until to do is resolved, and conflict as well.

File: code/controllers/configuration.dm [hunk 2]
```diff
@@ -40,7 +40,7 @@ var/list/gamemode_cache = list()
 	var/feature_object_spell_system = 0 //spawns a spellbook which gives object-type spells instead of verb-type spells for the wizard
 	var/traitor_scaling = 0 			//if amount of traitors scales based on amount of players
 	var/objectives_disabled = 0 			//if objectives are disabled or not
-	var/protect_roles_from_antagonist = 0// If security and such can be traitor/cult/other
+	var/protect_roles_from_antagonist = 1// If security and such can be traitor/cult/other
```
Comment (mattmatterson111) [inline code/controllers/configuration.dm:43]: Isn't that in the config.txt?

Comment (rhicora) [reply code/controllers/configuration.dm:43]: Yes, I recommend setting this in the config, but the default should be on.

Comment (JakeDaBoss) [reply code/controllers/configuration.dm:43]: I think that might be off from an earlier setting. Would explain a lot. I'll enable it if I find out it's off.

File: maps/site53/job/jobs.dm (hunk 1 above)
Comment (rhicora) [reply maps/site53/job/jobs.dm:113]: Task list has been taken care of, re-enabling these.

**Comment by rhicora** [comment]
Task list taken care of, doing a final test then this is ready to go.

**Comment by rhicora** [comment]
This is ready to go. @JakeDaBoss

**Review by JakeDaBoss — APPROVED** [review]

**Pull request merged**